.prettierrc

```diff
@@ -0,0 +1 @@
+"@wordpress/prettier-config"
```

Comment:
Do we needs to add these file in [.gitignore](https://github.com/WordPress/performance/blob/trunk/.gitignore)?

Reply:
No `.prettierrc` must not be ignored. It must be around for everyone.